src/index.html

```diff
@@ -11,7 +11,201 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+
+      <fieldset class="input__group">
+        <legend class="input__header">Personal information</legend>
+
+        <div class="input__one">
+          <label for="surname" class="input__label">Surname:</label>
```

Comment:
you don't have such class, remove it's usage everywhere
```suggestion
          <label for="surname">Surname:</label>
```